### DISCUSSION TITLE
Hallo

### DISCUSSION THREAD

**Discussion by Wildeswiesel** [Wildeswiesel/Space-Waffel#1] [General]
04.12.2023

- Testing Loops in Java; Phyton; C
- I set up my GitHub account and everything including that, like the codespaces template etc. 
- Created new Category "Blogs".
- Write my first Discussion. 
- opend a new Thread in Discord

**Comment by ActiveClientMods** [comment]
Hi mate, keep up the good work!

**Reply by HamidrezaRahimian** [reply]
was it easy for u to set u p jupyter notebook in the first day ?

**Reply by Wildeswiesel** [reply]
I don't think it was really that difficult and wbu?

**Comment by waldix04** [comment]
Hi, i think your First Discussion looks very nice. I hope you will make a good job.
